packages/keycloak-user-management/src/components/forms/UserForm/index.tsx

```diff
@@ -128,7 +128,7 @@ const UserForm: React.FC<UserFormProps> = (props: UserFormProps) => {
   return (
     <Row className="layout-content">
       {/** If email is provided render edit user otherwise add user */}
-      <h5 className="mb-3">{props.initialValues.email ? 'Edit User' : 'Add User'}</h5>
+      <h5 className="mb-3">{props.initialValues.email ? 'Edit' : 'Add'} User</h5>
```

Comment:
Let's  move these strings to the lang file

Reply:
Addressed

Reply:
we dont have lang file yet we are moving things like these into constants .. once MLS workflow is ready to implement we will switch over to use lang files